meson_options.txt

```diff
@@ -0,0 +1,2 @@
+option('selinux', type : 'combo', choices : ['auto', 'true', 'false'],
```

Comment:
Using `value: 'true` to be a default value would be better than having 3 choices

Reply:
This is copied straight from systemd. The had reasons for doing it this way.

Reply:
Defaulting it to true would cause meson to bail out with an error if libselinux is not installed and the user does not pass false explicitly.

Reply:
Ah i see, thanks for the explanation!

Reply:
No auto-detection please. This should be:

`option('selinux', type: 'boolean', value: false)`

We neither use whitespace in front of colons.